api-v3-beta.rst

```diff
@@ -0,0 +1,62 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
+putting the focus on the user cases we heard about from existing users.
+
+Our new API v3 has some big differences compared to the old one that make it more user-friendly and useful.
+
+These differences includes:
+
+* Authenticated only
+* Token based authentication
+* Read and Write
+* Project's slug resource based
+
+Allowing us to build new good features on top of them, like:
+
+* `Import a new project`_
+* `List all my projects`_
+* `Activate a version`_
+* `Trigger a build for a specific version`_
+* `Check build status`_
+* `Manage redirects`_ and `environment variables programmatically`_
```

Comment:
If the transition sentence above is in future tense (future tense as we are announcing beta availability currently), these should match or it sounds funny.

Reply:
I'm not sure to understand this comment.

Please, read my previous comment, though.

Reply:
I don't think this is really important, it seems to read okay either way.

My point was that we're discussing actions/queries "that *will be* available" -- that is, future tense. The bullet list could match tense there (so "Activating a version", "Triggering a build", etc).

However, reading now, present tense works just as well for the bullet list items.